model/test_results.go

```diff
@@ -581,20 +589,32 @@ func (t TestResult) getDuration() time.Duration {
 }
 
 func (t TestResult) convertToParquet() ParquetTestResult {
-	return ParquetTestResult{
-		TestName:        t.TestName,
-		DisplayTestName: utility.ToStringPtr(t.DisplayTestName),
-		GroupID:         utility.ToStringPtr(t.GroupID),
-		Trial:           utility.ToInt32Ptr(int32(t.Trial)),
-		Status:          t.Status,
-		LogTestName:     utility.ToStringPtr(t.LogTestName),
-		LogURL:          utility.ToStringPtr(t.LogURL),
-		RawLogURL:       utility.ToStringPtr(t.RawLogURL),
-		LineNum:         utility.ToInt32Ptr(int32(t.LineNum)),
-		TaskCreateTime:  utility.ToInt64Ptr(types.TimeToTIMESTAMP_MILLIS(t.TaskCreateTime.UTC(), true)),
-		TestStartTime:   utility.ToInt64Ptr(types.TimeToTIMESTAMP_MILLIS(t.TestStartTime.UTC(), true)),
-		TestEndTime:     utility.ToInt64Ptr(types.TimeToTIMESTAMP_MILLIS(t.TestEndTime.UTC(), true)),
+	result := ParquetTestResult{
+		TestName:       t.TestName,
+		Trial:          int32(t.Trial),
+		Status:         t.Status,
+		LineNum:        int32(t.LineNum),
+		TaskCreateTime: types.TimeToTIMESTAMP_MILLIS(t.TaskCreateTime.UTC(), true),
+		TestStartTime:  types.TimeToTIMESTAMP_MILLIS(t.TestStartTime.UTC(), true),
+		TestEndTime:    types.TimeToTIMESTAMP_MILLIS(t.TestEndTime.UTC(), true),
+	}
+	if t.DisplayTestName != "" {
+		result.DisplayTestName = utility.ToStringPtr(t.DisplayTestName)
 	}
+	if t.GroupID != "" {
```

Comment:
What's this change about as well?

Reply:
Yeah these fields are optional and not always populated.